README.md

```diff
@@ -95,10 +97,35 @@ After your Docker container successfully builds and is running locally you can s
 3. Inside the `launch.json` file, set the `configuration->service->serviceAccountName` attribute to the service account email you just created.
 
 ## Deploying your own instance with terraform
-To run the pipeline with terraform, create your own `terraform.tfvars` file in the same directory as the other terraform files. For each variable declared in `prototype_variables.tf` that doesn't have a default, add your own for testing. Typically your own variables should be unique and can just be prefixed with your name or ldap. There are some that have specific requirements like project ids, code paths, and image paths.
-
-Currently the setup deploys both a data ingestion cloud funtion and a data ingestion cloud run instance. These are duplicates of each other. Once we get the cloud run one fully working we will delete the cloud fuction, but for now if you want to not have a duplicate while developing you can just comment out the setup for whichever one you don't want to use in `prototype.tf`
-
-Note that currently terraform doesn't diff the contents of the functions/cloud run service, so to get them to redeploy you can either
+Before deploying, make sure you have installed Terraform and a Docker client (e.g. Docker Desktop). See [One time setup](#one-time-setup) above.
+1. Create your own `terraform.tfvars` file in the same directory as the other terraform files. For each variable declared in `prototype_variables.tf` that doesn't have a default, add your own for testing. Typically your own variables should be unique and can just be prefixed with your name or ldap. There are some that have specific requirements like project ids, code paths, and image paths.
```

Comment:
Can you specify which of these steps need to be done just once vs every time you want to deploy new code?